src/main/kotlin/frc/robot/lib/ColorUtils.kt

```diff
@@ -1,21 +1,20 @@
 package frc.robot.lib
 
-import com.pathplanner.lib.util.FlippingUtil
 import edu.wpi.first.math.geometry.Pose2d
 import edu.wpi.first.math.geometry.Translation2d
 import frc.robot.IS_RED
 
 fun getTranslationByColor(translation: Translation2d): Translation2d {
     return if (IS_RED) {
-        FlippingUtil.flipFieldPosition(translation)
+        translation.flip()
     } else {
         translation
     }
 }
```

Comment:
You can make this much shorter:
```suggestion
fun getTranslationByColor(translation: Translation2d) = if (IS_RED) translation.flip() else translation
```